frontend/src/views/Contacts.vue

```diff
@@ -1,7 +1,87 @@
 <template>
-  <div>
-    <h1>Contact</h1>
-    <p>Contacts part of the application</p>
-  </div>
+  
+    <div class="contactUs">
+      <div class="contact__content">
+         <div class="contactUs__text"><h1>Do you have questions?<br> Contact Us:</h1></div>
+    <div class="contactUs__img"><img src="../assets/img/5d8d941e933874bb0b403ffc5ac47e08.jpeg" alt="img"></div>
+      </div>
+     
+    
+     <div class="contactUs__page">
+       <p class="contact__text">We want to hear you! 
+         <br>
+         <br>
+
+         Kindly reach out to us via our CONTACT FORM or any of our media channels below
+</p>
+
+        <form class="form" @submit.prevent="sendEmail">
+         
+         
+          <input 
+            type="text" 
+            v-model="name"
+            name="name"
+            placeholder="Your Name"
+            required
+          />
+         
+          <input 
+            type="email" 
+            v-model="email"
+            name="email"
+            placeholder="Your Email"
+            required
+            />
+         
+          <textarea 
+            name="message"
+            v-model="message"
+            cols="30" rows="5"
+            placeholder="Message">
+          </textarea>
+          
+          <input type="submit" value="Send">
+        </form>
+     </div>
+    </div>
+  
 </template>
-<style></style>
+
+<script>
+import emailjs from 'emailjs-com';
```

Comment:
`import emailjs from '@emailjs/browser';`

the previous version is deprecated..also check the documentation, they even have an example for Vue
https://www.emailjs.com/docs/examples/vuejs/

Reply:
Yep i saw ,and in fact first pullrequest was based on this template,but to change style was difficult for me with that.Thats why i changed that.ans i see email from u.